src/components/dashboard/main-content/course-enrollments/CourseEnrollments.jsx

```diff
@@ -134,7 +140,8 @@ export class CourseEnrollments extends Component {
           courseRuns={courseRuns.upcoming}
         />
         <CourseSection
-          title="Completed courses"
+          title="Archived courses"
+          subtitle={ARCHIVED_COURSES_SECTION_SUBTITLE}
           component={CompletedCourseCard}
           courseRuns={courseRuns.completed}
```

Comment:
I totally agree. I think we should rename complete to archive going forward. In this PR I am accepting a little bit of duplication fo code also actually. My plan is to create a second PR which will include

* renaming complete to archive
* address some of the (intentional) duplication/DRY issues you will see in this PR. I thought it was better to have a new modal workflow since resuing the existing modal was leading to a lot of kung fu I would have to do in text names, constants, method names etc.

thoughts?

Reply:
The duplication you speak of - if you're referring to the ModalBody and ModalError and the like, I mean I imagine we could maybe refactor that to move out of the mark-complete-modal directory and have it take in props to indicate if the verbiage should reference completion or archiving.

But otherwise, I don't personally see it as a big deal, but I haven't dived into the front end aspects of these code bases too much, so I'm sure others may have stronger opinions.

Reply:
Yeah, there's an opportunity to reduce duplication between the MarkCompleteModal and UnarchiveModal components. Not a high priority issue, though so probably doesn't make sense to include as part of this work.

Reply:
resolving for this PR but I totally on board with some of the thoughts here.. I think we can cleanup some of the code for sure.. esp. around duplication between 'archive' and 'mark complete' course workflows as well as some language/naming cleanup. I jsut did not want to include a bigger change in this PR